azurelinuxagent/common/cgroupapi.py

```diff
@@ -556,41 +564,37 @@ def create_and_start_unit(unit_filename, unit_contents):
         except Exception as e:  # pylint: disable=C0103
             raise CGroupsException("Failed to create and start {0}. Error: {1}".format(unit_filename, ustr(e)))
 
+    @staticmethod
+    def _get_azure_slice_name():
+        return "{0}.slice".format(AZURE_CGROUP_NAME)
+
     @staticmethod
     def _get_extensions_slice_root_name():
-        return "system-{0}.slice".format(EXTENSIONS_ROOT_CGROUP_NAME)
+        return "{0}-{1}.slice".format(AZURE_CGROUP_NAME, EXTENSIONS_ROOT_CGROUP_NAME)
```

Comment:
In cgroups, if you name a slice `A-B.slice`, it means slice `B` will be placed under parent slice `A`.